contracts/ETO/ETOCommitment.sol

```diff
@@ -499,21 +503,20 @@ contract ETOCommitment is
             bool isEligible,
             uint256 minTicketEurUlps,
             uint256 maxTicketEurUlps,
-            uint256 equityTokenInt,
+            uint256 equityTokenAmount,
             uint256 neuRewardUlps,
-            bool maxCapExceeded,
-            uint256 spentAmountEurUlps
```

Comment:
this should probably not have been removed

Reply:
it was totally caught by the test :) example: `ETOCommitment.js` `it("should calculate contribution", async () => {`

I removed it because for decimals 18 tokens this value is not relevant - will be 1 Ulps max. When calculating token amount we round down - for indivisible tokens you could buy whole tokens only. And difference between amount sent to the contract and amount spent could be quite big (up to token price!)

there's plenty comments on that in `ETOTerms::calculateContribution`